apps/web/src/scenes/CommunityPage/CommunityPageTabs.tsx

```diff
@@ -1,18 +1,52 @@
-import { useCallback } from 'react';
+import { useCallback, useMemo } from 'react';
+import { graphql, useFragment } from 'react-relay';
 import styled from 'styled-components';
 
 import { HStack } from '~/components/core/Spacer/Stack';
 import { BaseXL } from '~/components/core/Text/Text';
+import { CommunityPageTabsFragment$key } from '~/generated/CommunityPageTabsFragment.graphql';
 import colors from '~/shared/theme/colors';
 
 import { CommunityPageTab } from './CommunityPageView';
-
 type Props = {
   onSelectTab: (value: CommunityPageTab) => void;
   activeTab: CommunityPageTab;
+  communityRef: CommunityPageTabsFragment$key;
 };
 
-export default function CommunityPageTabs({ onSelectTab, activeTab }: Props) {
+export default function CommunityPageTabs({ onSelectTab, activeTab, communityRef }: Props) {
+  const community = useFragment<CommunityPageTabsFragment$key>(
+    graphql`
+      fragment CommunityPageTabsFragment on Community {
+        posts(before: $communityPostsBefore, last: $communityPostsLast)
+          @connection(key: "CommunityFeed_posts") {
+          edges {
+            node {
+              ... on Post {
+                __typename
+              }
+            }
+          }
+        }
+      }
+    `,
+    communityRef
+  );
+
+  const feedData = useMemo(() => {
+    const events = [];
+
+    for (const edge of community.posts?.edges ?? []) {
+      if (edge?.node?.__typename === 'Post' && edge.node) {
+        events.push(edge.node);
+      }
+    }
+
+    return events;
+  }, [community.posts?.edges]);
+
+  const totalPosts = feedData?.length;
```

Comment:
we should use pageInfo.total from the connection instead , that will show the accurate total unaffected by pagination limits

Reply:
oh yeah.. right right!